test/Microsoft.AspNet.Razor.Test/TestFiles/CodeGenerator/CS/Source/NullPropagationExpressions.cshtml

```diff
@@ -0,0 +1,13 @@
+﻿@{
+    @ViewBag?.Data
+    @ViewBag.Method?(Hello World)
```

Comment:
Has support for `Method?(...)` been added to the language recently?  Null-conditional syntax previously included only properties (`?.Property`) and indexers (`?[index]`).